src/psyclone/dynamo0p3.py

```diff
@@ -8495,19 +8491,21 @@ def intent(self):
                  by the kernel argument metadata
         :rtype: str
         '''
-        if self.access == "gh_read":
+        if self.access == AccessType.READ:
```

Comment:
Please simplify the `elif`s to `if`s where possible to keepy pylint happy.

Reply:
Done.